index.bs

```diff
@@ -992,7 +996,7 @@ The <dfn attribute for=ServiceWorkerGlobalScope>cookieStore</dfn> getter steps a
 
 [=Cookie=] attribute-values are stored as [=byte sequences=], not strings.
 
-To <dfn>encode</dfn> a |string|, run [=UTF-8 encode=] on |string|.
+To encode a |string|, run [=UTF-8 encode=] on |string|.
```

Comment:
Maybe we don't need this any more, if you're getting an unused definition warning?

Reply:
Indeed

```
WARNING: Unexported dfn that's not referenced locally - did you mean to export it?
      <dfn data-dfn-type="dfn" id="encode" data-lt="encode" data-noexport="by-default">encode<a href="#encode" class="self-link"></a></dfn>
```